keyboards/sol/common/knob_v2.c

```diff
@@ -0,0 +1,71 @@
+#include "knob_v2.h"
```

Comment:
If this is for the rotary encoder, you may not need this. 

https://docs.qmk.fm/#/feature_encoders

And it's handled by the matrix init and matrix scan functions, so a custom matrix shouldn't prevent it from working